tasks/postgres-cluster.yml

```
@@ -22,6 +22,12 @@
    - ansible_distribution_release != 'NA'
    - postgres_version in ['9.3', '9.4', '9.5', '9.6', '10', '11', '12']
 
+- name: Install pglogical version {{ postgres_version }}
+  apt: name=postgresql-{{ postgres_version }}-pglogical
```

could you use the normal yaml syntax to pass parameters to the `apt` module please?

```suggestion
  apt: 
    name: "postgresql-{{ postgres_version }}-pglogical"
```

---

Thanks, applied :+1: